experimental/delegate/.gitignore

```diff
@@ -0,0 +1,2 @@
+*.vmfb
```

Comment:
Is this intentional?

Reply:
Yes, the demos are designed to run from the delegate directory for now.  I intend for those build artifacts not to show up in git status, etc., for any potential users

Reply:
It would be good if the way this is done, it just either uses a tmp directory or it generates in the working directory. Polluting the codespace isnt great. But can be a follow up.

Reply:
It generates in the cwd already, and no automation generates these files, so you get pollution only if you manually ask for it!  :-) .  You can output to a different dir, or run the demo from a different cwd if you'd rather.

Personally, for demos in a source tree, because the demo inputs are there, I find it easiest to generate my outputs there as well, as long as they don't show up in my git status.  I have also observed that most developers prefer to operate this way, probably because we're too lazy to make a copy of the demo tree or work with separate input and output dirs.

In any case, when I get around to making e2e tests for CI, I'll be sure not to pollute the source tree.  I agree that the build process should never create any artifacts mixed in with the source.